applications/vasp/Performance Study 2/benchmarks/README.md

```diff
@@ -0,0 +1,12 @@
+# VASP Benchmarking
+
+talk about goals of study, different input systems, systems run on, calculation performed (energy calculation), modules and version of VASP used
```

Comment:
These look like notes, did you intend to put different text here?

Reply:
file shouldn't be there, deleted file